lib/server.js

```diff
@@ -62,10 +62,15 @@ function prettyPrintErrors(request, reply) {
 module.exports = (config) => {
     // Hapi Cross-origin resource sharing configuration
     // See http://hapijs.com/api for available options
+
+    let corsOrigins = [config.ecosystem.ui];
+
+    if (Array.isArray(config.ecosystem.allowCors)) {
+        corsOrigins = corsOrigins.concat(config.ecosystem.allowCors);
+    }
+
     const cors = {
-        origin: [
-            config.ecosystem.ui
-        ],
+        origin: corsOrigins,
```

Comment:
and here just set it to config.ecosystem.allowCors

Reply:
This way we will be forcing users to repeat value of ecosystem.ui again here. That's just duplication and won't be backward compatible.